### PR TITLE
generate publicize plugin configuration file

### PR DESCRIPTION
**Pull request by droslean**
Generates the publicize plugin configuration file.

/cc @openshift/openshift-team-developer-productivity-test-platform 

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: clusters/app.ci/publicize/_config.yaml [hunk 1]
```diff
@@ -0,0 +1,181 @@
+repositories:
+  openshift-priv/ansible-service-broker: openshift/ansible-service-broker
```
Comment (alvaroaleman) [inline clusters/app.ci/publicize/_config.yaml:2]: @droslean why is this list btw needed, we can just derive the one from the other or not?

Comment (droslean) [reply clusters/app.ci/publicize/_config.yaml:2]: This is a configuration file for the publicize plugin. see https://github.com/openshift/ci-tools/pull/872

Comment (alvaroaleman) [reply clusters/app.ci/publicize/_config.yaml:2]: Yeah but why does the publicize plugin not simply infer the public name from the private name by stripping `-priv` from the org? Then we save ourselves the configfile, the tool to update the configfile and the periodic.

Comment (droslean) [reply clusters/app.ci/publicize/_config.yaml:2]: Because the org can be different than `openshift`.

**Pull request merged**